### DISCUSSION TITLE
PxFEMCloth

### DISCUSSION THREAD

**Discussion by Gloukopoulos** [NVIDIA-Omniverse/PhysX#319] [Q&A] [answered]
Hello, I’ve been experimenting with PhysX SDK and I noticed that there is an under construction feature which has to do with FEM cloth simulation. Is there any way that I can actually give it a try on this current situation of being under construction issue or if not, when should we expect this feature to be released ?

p.s. I have noticed that the changelog mentions something about using PxFEMCloth but I haven;t yet figured out how to use it

**Comment by sschirm** [comment, marked as answer]
Hi @Gloukopoulos , 
It's been a feature under construction, so not really available. Soon it will be available as PxDeformableSurface, but still quite fresh and we are still working on it.

**Reply by Gloukopoulos** [reply]
Great, thanks a lot for your reply !
